_spinnaker_install_admin_guides/install-on-aws-halyard-pod.md

```diff
@@ -0,0 +1,668 @@
+
+## Create an IAM user, using an inline policy
+
+You can create an IAM user with credentials, and provide that to Spinnaker via Halyard
+
+1. Log into the AWS Console (Web UI)
+2. Navigate to the IAM Console (Click on "Services" at the top, and then on "IAM" under "Security, Identity, & Compliance")
+3. Click on "Users" on the left
+4. Click on "Add user"
+5. Give your user a distinct name, per your organization's naming conventions. For this document, we will use `s3-spinnaker-abcxyz`
+6. Click on "Programmatic access"
+7. We will not be adding a distinct policy to this user. Click on "Next: Tags". _You may receive a warning about how there are no policies attached to this user - this warning can be ignored._
+8. Optionally, add tags, then click on "Next: Review"
+9. Click "Create user"
+10. Save the Access Key ID and Secret Access Key - these will be used later, during Halyard configuration
+11. Click "Close"
+
+Then, add an inline policy to your IAM user:
+
+1. Click on your newly-created IAM user
+2. Click on "Add inline policy" (on the right)
+3. Click on the "JSON" tab
+4. Add this text (replace `s3-spinnaker-abcxyz` with the name of your bucket)
+
+	```json
+	{
+	  "Version": "2012-10-17",
+	  "Statement": [
+	    {
+	      "Effect": "Allow",
+	      "Action": "s3:*",
+	      "Resource": [
+	        "arn:aws:s3:::spinnaker-abcxyz",
+	        "arn:aws:s3:::spinnaker-abcxyz/*"
+	      ]
+	    }
+	  ]
+	}
+	```
+
+5. Click on "Review Policy"
+
+6. Give your inline policy some name. For example `s3-spinnaker-abcxyz`
+
+7. Click "Create Policy"
+
+## Create an IAM policy attached to the Kubernetes nodes, using an inline policy
+
+Alternately, you can attach an IAM policy to the role attached to your Kubernetes nodes.
+
+1. Log into the AWS Console (Web UI)
+2. Navigate to EC2 (Click on "Services" at the top, and then on "EC2" under "Compute")
+3. Click on one of your Kubernetes nodes
+4. In the bottom section, look for "IAM role" and click on the role
+5. Click on "Add inline policy" (on the right)
+6. Click on the "JSON" tab
+7. Add this text (replace `s3-spinnaker-abcxyz` with the name of your bucket)
+
+	```json
+	{
+	  "Version": "2012-10-17",
+	  "Statement": [
+	    {
+	      "Effect": "Allow",
+	      "Action": "s3:*",
+	      "Resource": [
+	        "arn:aws:s3:::spinnaker-abcxyz",
+	        "arn:aws:s3:::spinnaker-abcxyz/*"
+	      ]
+	    }
+	  ]
+	}
+	```
+
+8. Click on "Review Policy"
+
+9. Give your inline policy some name. For example `s3-spinnaker-abcxyz`
+
+10. Click "Create Policy"
+
+# Start the Halyard container
+
+Halyard is a tool used to install Spinnaker (it generates Kubernetes manifests for each of the Spinnaker services).  First, create a namespace for Spinnaker to run in:
+
+```bash
+# Feel free to use a different namespace for this
+kubectl create ns spinnaker
```

Comment:
may need to use the kubectl --kubeconfig 
based on the kubeconfig-aws (as mentioned earlier)

Reply:
removed the reference to kubeconfig across the board